src/subscript/runrms/runrms.py

```diff
@@ -259,10 +289,31 @@ def do_parse_args(self, args):
             args = sys.argv[1:]
 
         myparser = get_parser()
+        parsed_args = myparser.parse_args(args)
+
+        # just 'runrms' shall not trigger listing, hence test of len(inargs)
+        if (
+            parsed_args.rversion is None
+            and len(args) > 0
+            and ("-v" in args or "--version" in args)
+        ):
+            parsed_args.listversions = True
+            self.warn_empty_version = True
+
+        if parsed_args.project is None and parsed_args.project_alt:
+            parsed_args.project = parsed_args.project_alt
+
+        if parsed_args.project and parsed_args.project_alt:
+            xwarn("Conflict use two types of project input. First argument will win!")
```

Comment:
Should this not be a hard error?

Reply:
I think this is buggy, you will trigger this warning when doing only `runrms --project foo`

Reply:
Corrected to elif on line 306

Reply:
Can keep as a soft warning